lesson18/script.js

```diff
@@ -139,6 +139,10 @@ const addClickEventForPagination = (sliderData) => {
     const arrayPaginations = [...document.getElementsByClassName("js-pagination__item")];
     
         paginations.addEventListener("click",function(e){
+            //イベントハンドラを登録した要素は何もしない
+            if(e.currentTarget === e.target){
+                return;
+            }
             sliderStatus.currentIndex = arrayPaginations.indexOf(e.target);
 
             switchSlider(sliderData);
```

Comment:
@sae-github @haru-programming 
Thank you for teaching me a lot of about event delegation.

I added a process that does nothing when the element in which the event handler is registered, 
e.currentTarget is `paginations` here.